tests/plugins/test_tricritical_review_evidence.py

```
@@ -568,101 +575,247 @@ def rolecasting(
+                "evidence": identity("product-attestation-v1", role),
+            }
+            assurance_minimum = {
+                field: "product-attested"
+                for field in (
+                    "target",
+                    "model",
+                    "topology",
+                    "authority",
+                    "execution_result",
+                )
+            }
```

Ruff (0.16.2)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)